packages/hooks/src/useAccountStatus.ts

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from 'react';
+import { useFetch } from '@deriv/api';
+
+const useAccountStatus = () => {
+    const { data, ...rest } = useFetch('get_account_status');
+
+    const account_status = useMemo(() => data?.get_account_status, [data?.get_account_status]);
+    return {
+        data: account_status,
+        ...rest,
+    };
+};
+
+export default useAccountStatus;
```

Comment:
This hook can be replaced by `useGetAccountStatus` in `@deriv/api` 🙏🏻

Reply:
This wasn't there in master when @shahzaib-deriv started working on it. Will replace this